packages/cli/src/databases/entities/test-entity.ts

```diff
@@ -0,0 +1,44 @@
+import {
+	Column,
+	Entity,
+	Generated,
+	Index,
+	ManyToOne,
+	OneToOne,
+	PrimaryColumn,
+	RelationId,
+} from '@n8n/typeorm';
+import { Length } from 'class-validator';
+
+import { AnnotationTagEntity } from '@/databases/entities/annotation-tag-entity.ee';
+import { WorkflowEntity } from '@/databases/entities/workflow-entity';
+
+import { WithTimestamps } from './abstract-entity';
+
+@Entity()
+@Index(['workflow'])
+@Index(['evaluationWorkflow'])
+export class TestEntity extends WithTimestamps {
```

Comment:
Test is very overloaded term. Could we use something more specific? E.g. `WorkflowTest`? Also could we add a short description for this entity

Reply:
I don't know what we should name this, but I'm certain that calling this `Test*` would be confusing for many people.

Even `WorkflowTest` is overloaded, as it can already either refer to the special test type we have `nodes-base`, as well as the [test-workflows repo](https://github.com/n8n-io/test-workflows).